test/Domain/LeanCode.DomainModels.Tests/LeanCode.DomainModels.Tests.csproj

```diff
@@ -4,8 +4,40 @@
     <Nullable>enable</Nullable>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp" PrivateAssets="all" />
+  </ItemGroup>
+
   <ItemGroup>
     <ProjectReference Include="../../../src/Domain/LeanCode.DomainModels/LeanCode.DomainModels.csproj" />
+    <ProjectReference
+      Include="../../../src/Domain/LeanCode.DomainModels.Generators/LeanCode.DomainModels.Generators.csproj"
+      OutputItemType="Analyzer" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NETCore.App.Ref" Version="7.0.0" IncludeAssets="none" PrivateAssets="all" GeneratePathProperty="true" />
```

Comment:
`GeneratePathProperty` allows you to find the path using properties named like `Pkg${PACKAGE//./_}` instead of accessing `$(NuGetPackageRoot)`, we should probably do that.